english/london/london.md

```diff
@@ -6,3 +6,4 @@ I created a new folder and file within it (this one)
 I'm now going to try commit these changes and send a pull request - hopefully
 Hey! You got this! You are now ready to help people build open-source software!
 AWESOME!
+Similar to you I have no clue what is going on... but I love [Dragonball Z](../Dragonball-Z/Dragonball-Z.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.